### ISSUE TITLE
move all SGCs into the relevant week's folder

### ISSUE THREAD

**Issue by jsms90** [foundersandcoders/master-reference#637]
Would it not be easier for the week facilitators and students to review all the feedback (SGCs) for a particular week by having it all in one place?

i.e.
```
|- week-1
|- |- feedback
|- |- |- FAC11-cohort-sgc.md
|- |- |- FAC11-mentor-sgc.md
|- |- |- FACG1-cohort-sgc.md
|- |- |- FACG1-mentor-sgc.md
|- |- |- FACN1-cohort-sgc.md
|- |- |- FACN1-mentor-sgc.md
|- |- README.md
|- |- learning-outcomes.md
|- |- project.md
|- |- research-afternoon.md
|- |- resources.md
```

@des-des @sohilpandya @oliverjam @ronanyeah @mantagen @bo-bok @mattlub @ZooeyMiller @yvonne-liu @finnhodgkin @m4v15 

+ [x] move mentor retrospectives into week folder (e.g. `curriculum-planning/postgres` into `week-6/feedback`
+ [x] move mentor SGCs during FAC10 from `london-programme` into `master-reference`
+ [ ] move cohort SGCs into week folder (e.g. `london-programme/stop-go-continues/fac-11/cohort-week-1.md` becomes `master-reference/coursebook/week-1/feedback/fac11-cohort-sgc.md`)

**Comment by mattlub** [comment]
I agree in that finding them would be easier but most of what comes up in SGC is not really related to the curriculum, at least in Nazareth 1 & 2

**Comment by des-des** [comment]
@jsms90 I know this is fairly tangential, but right now looking at pull requests that are not directly editing the content of the master reference is fairly time consuming, could we have another repo? Called like master-reference-meta and minutes.

That way we have a clear record with version control. Right now we have a set of contributing rules designed to ensure quality of curriculum materials. My fear is that if the majority of maintenance of this repo is doing things like approving minutes from meetings you were not involved in, it will lead to maintainer disengagement.

**Comment by jsms90** [comment]
@des-des I reckon we should just be pinging the relevant people in different PRs, rather than distributing all the content that relates to the curriculum in different places? :woman_shrugging:

**Comment by ZooeyMiller** [comment]
I would agree with @mattlub. SGCs would be easier to find in the MR but I don't necessarily think they belong here. A lot of the stuff in SGCs ends up being environment or interpersonal based which I don't really think that belongs in the MR.

**Comment by jsms90** [comment]
Yeah it's true. But a lot is about the curriculum itself as well.

And at the end of the day, this is our evidence for what each cohort actually thinks about the curriculum. Isn't that how you justify whether a change should be made or not? You have a look into the common patterns that keep recurring, rather than just going off personal opinion?

e.g.
https://github.com/foundersandcoders/master-reference/issues/581
https://github.com/foundersandcoders/master-reference/issues/587
https://github.com/foundersandcoders/master-reference/issues/577

**Comment by jsms90** [comment]
Perhaps this comes from the fact that, when I've been facilitating SGCs with the students, I've been prompting them with:
+ Imagine when you'll be running this week yourself, what do you want to record now, to remind yourself to do / not do for the next cohort?
+ Imagine what you would want to say to a mentor who is about to run this in Nazareth. What advice would you give them, based on your experience?
+ What has this week taught you that you want the mentors for next week to know about? What can we do to make next week better for you?

So a lot of what is brought up in London is directly related to the curriculum & I think it would be much easier for people to reflect if they had what the London mentors said about the week sitting right next to what they said about the week.

But it's true that they also mention things all the time which are location-specific i.e. things about the environment (the toilet, the door, ordering coffee, the towels). But tbh, even with the environmental things, you can notice patterns across all campuses that could inform how you choose to run a FAC curriculum anywhere. For example, the organisation of the tables in the room, the lack of posters on the walls, etc.

At the very least, can we move the mentor retrospectives into the particular week folders, so that all I have to do is point the students to that folder?

**Comment by jsms90** [comment]
Given that we're asking students to review the stop go continues during curriculum planning, it still seems like it would be worth having them in the same place.

Yesterday, to save the students time, I added links to the top comment of each `week-epic` issue, which linked them directly to the stop go continues. This extra little bit of admin would be easily avoided by having all of the SGCs from all campuses sitting inside the `week-x` folder.

But even if we don't agree on putting the student SGCs into `master-reference`, it would still help to move the mentor retrospectives (which are definitely about nothing but the curriculum) inside each `week-x` folder, so that this folder contains everything the students need to review.

I've opened a PR @des-des @mantagen @m4v15. Let me know what you think.
